CHANGELOG.md

```diff
@@ -1,5 +1,262 @@
+- Fix rn-cli linting issues (#22099) ([7b10a02](https://github.com/facebook/react-native/commit/7b10a02) by [@ignacioola](https://github.com/ignacioola))
+- TouchEventEmitter: Fix assignment of Y coordinates (#22160) ([6b6a27c](https://github.com/facebook/react-native/commit/6b6a27c) by [@empyrical](https://github.com/empyrical))
+- Fix inline styles in IntegrationTests (#22165) ([1d62e94](https://github.com/facebook/react-native/commit/1d62e94) by [@ignacioola](https://github.com/ignacioola))
+- Fix inline styles warning in Libraries (#22161) ([41eb2da](https://github.com/facebook/react-native/commit/41eb2da) by [@ignacioola](https://github.com/ignacioola))
+- Fix build error caused by -Werror=class-memaccess (#823) ([31439f8](https://github.com/facebook/react-native/commit/31439f8) by [@hooddanielc](https://github.com/hooddanielc))
+- Fix IllegalArgumentException when dismissing ReactModalHostView ([e57ad4e](https://github.com/facebook/react-native/commit/e57ad4e) by [@mdvacca](https://github.com/mdvacca))
+- Fix internal types on top of TextInput refactor ([ad7d8f8](https://github.com/facebook/react-native/commit/ad7d8f8) by [@TheSavior](https://github.com/TheSavior))
+- Fix inline styles eslint warnings for examples (#22123) ([7b3c91e](https://github.com/facebook/react-native/commit/7b3c91e) by [@ignacioola](https://github.com/ignacioola))
+- Fix ReactInstanceManager deadlock ([df7e8c6](https://github.com/facebook/react-native/commit/df7e8c6) by [@ayc1](https://github.com/ayc1))
+- Fix ReactRootView attachRootView race condition ([be282b5](https://github.com/facebook/react-native/commit/be282b5) by [@ayc1](https://github.com/ayc1))
+- UITemplateProcessor: Fix case of include path (#22239) ([0436bfc](https://github.com/facebook/react-native/commit/0436bfc) by [@empyrical](https://github.com/empyrical))
+- Fix regression in StyleSheet.setStyleAttributePreprocessor (#22262) ([0408533](https://github.com/facebook/react-native/commit/0408533) by [@brentvatne](https://github.com/brentvatne))
+- Fix crash when releasing RN views ([83405ff](https://github.com/facebook/react-native/commit/83405ff) by [@ayc1](https://github.com/ayc1))
+- Fix crash when removing root nodes ([b649fa9](https://github.com/facebook/react-native/commit/b649fa9) by [@ayc1](https://github.com/ayc1))
+- Fix React Native AsyncMode and DevTools ([aacb06c](https://github.com/facebook/react-native/commit/aacb06c) by [@bvaughn](https://github.com/bvaughn))
+- reapply TextInput es6 conversion with fixes, attemps to fix ([9ea1295](https://github.com/facebook/react-native/commit/9ea1295) by [@sahrens](https://github.com/sahrens))
+- Back out "reapply TextInput es6 conversion with fixes, attemps to fix" ([6f34bc4](https://github.com/facebook/react-native/commit/6f34bc4) by [@sahrens](https://github.com/sahrens))
```

Comment:
This and the one above it should be removed since they are the commit and revert